starters/nuxt3-pinia-vuetify/components/CounterExample/CounterExample.test.ts

```diff
@@ -0,0 +1,88 @@
+import { describe, expect, test } from 'vitest'
+import { createTestingPinia } from '@pinia/testing'
+import { fireEvent, render, screen } from '../../test/utils'
```

Comment:
Lets change this import to an alias instead!
```suggestion
import { fireEvent, render, screen } from '@/components/test/utils'
```